appveyor.yml

```diff
@@ -0,0 +1,15 @@
+image: Visual Studio 2017
+
+platform:
+  - x64
```

Comment:
Can we also do 32bit builds on Windows? On macOS and Linux I´m ok with just providing 64bit builds, but there are many old Windows machines out there...

Reply:
This is the version of the os image to be used. (Because we can build for both on 64-bit)
It is currently building one `.exe` file for both 32-bit and 64-bit version.

Reply:
Excellent, thanks!